src/Auth_Command.php

```diff
@@ -523,6 +535,12 @@ private function get_auths( $site_url, $user, $error_if_empty = true ) {
 	 *     # Remove global auth on all sites (but not admin tools) with default username(easyengine)
 	 *     $ ee auth delete global
 	 *
+	 *     # Remove whitelisted IPs on site
+	 *     $ ee auth delete example.com --ip
```

Comment:
Wrong example. Need to provide value in `--ip` flag. Boolean flag is not excepted here.
Either change the example or you can update the delete logic.

Reply:
Done